Automattic/wp-calypso
# Support Lib: Update cancelation support URL to reflect correct anchor

#### Changes proposed in this Pull Request

* Updates the URL to the manage and cancel plans/domains FAQ to use the correct in-page anchor.

**Before**

<img width="953" alt="Screen Shot 2020-07-16 at 9 54 58 AM" src="https://user-images.githubusercontent.com/2124984/87680320-45752080-c74b-11ea-8954-75e7104b3fe8.png">

**After**

<img width="961" alt="Screen Shot 2020-07-16 at 9 52 41 AM" src="https://user-images.githubusercontent.com/2124984/87680331-4a39d480-c74b-11ea-8841-119ea9436698.png">

#### Testing instructions

* Switch to this PR
* Add a plan to your cart and go to checkout
* Look at the fine print; the link for "how to cancel" should point here: https://wordpress.com/support/manage-purchases/#faq-cancelling-your-plans-and-domains; click the link to make sure it works correctly.
